flink-table/flink-sql-client/src/main/java/org/apache/flink/table/client/gateway/local/ResultStore.java

```diff
@@ -63,7 +63,7 @@ public ResultStore(Configuration flinkConfig) {
 
 		if (env.getExecution().inStreamingMode()) {
 			// determine gateway address (and port if possible)
-			final InetAddress gatewayAddress = getGatewayAddress(env.getDeployment());
+			InetAddress gatewayAddress = getGatewayAddress(env.getDeployment());
```

Comment:
why deleting final?

Reply:
I think it's unintentional change,  will revert it.